src/main/java/com/gtnewhorizons/angelica/mixins/Mixins.java

```diff
@@ -319,12 +318,17 @@ public enum Mixins {
             "base.MixinSimpleReloadableResourceManager",
 
             "base.MixinMinecraft"
-
-            // TODO merge renderpass changes into Sodium renderer
-            // "renderpass.MixinEntityRenderer",
-            // "renderpass.MixinRenderBlocks",
-            // "renderpass.MixinRenderGlobal",
-            // "renderpass.MixinWorldRenderer"
+        ))
+    ),
+    MCPATCHER_FORGE_RENDERPASS(new Builder("MCPatcher Forge Renderpass")
+        .setPhase(Phase.EARLY)
+        .setApplyIf(() -> NotFineConfig.renderPass)
```

Comment:
Is this the right config?

Reply:
Yeah. I plan to migrate everything on NotFine's end into one config so stuff isn't as spread out. I just put it there directly since it doesn't fall into one of the categories already in the MCPF config.